tests/dualtor/test_t1_to_standby.py

```diff
@@ -0,0 +1,82 @@
+"""
+Send traffic destined to server under standby mux and verify the behavior of tunnel, ecmp, etc.
+"""
+import contextlib
+import logging
+import pytest
+import pdb
+import random
+
+from ptf import testutils
+from tests.common.dualtor.dual_tor_mock import *
+from tests.common.dualtor.tunnel_traffic_utils import tunnel_traffic_monitor
+from tests.common.dualtor.dual_tor_utils import get_t1_ptf_ports
+
+
+pytestmark = [
+    pytest.mark.topology("t0")
+]
+
+
+@pytest.fixture(scope="function")
+def build_packet_to_server(select_test_interface, ptfadapter, tor, tunnel_traffic_monitor):
+    """Build packet destinated to server selected by test_interface."""
+    _, server_ipv4 = select_test_interface
+    pkt_dscp = random.choice(range(0, 33))
+    pkt_ttl = random.choice(range(3, 65))
+    pkt = testutils.simple_ip_packet(
+        eth_dst=tor.facts["router_mac"],
+        eth_src=ptfadapter.dataplane.get_mac(0, 0),
+        ip_src="1.1.1.1",
+        ip_dst=server_ipv4,
+        ip_dscp=pkt_dscp,
+        ip_ttl=pkt_ttl
+    )
+    logging.info("the packet destinated to server %s:\n%s", server_ipv4, tunnel_traffic_monitor._dump_show_str(pkt))
+    return pkt
+
+
+def show_arp(duthost, neighbor_addr):
+    """Show arp table entry for neighbor."""
+    command = "/usr/sbin/arp -n %s" % neighbor_addr
+    output = duthost.shell(command)["stdout"].splitlines()
+    if "no entry" in output[0]:
+        return {}
+    headers = ("address", "hwtype", "hwaddress", "flags", "iface")
+    return dict(zip(headers, output[1].split()))
+
+
+@contextlib.contextmanager
+def flush_neighbor(duthost, neighbor):
+    """Flush neighbor entry for server in duthost."""
```

Comment:
Can you please add an argument ```restore=True``` ? If ```restore == False```, then no neigh is recovered at exit. Some cases will verify the neigh can be relearnet after flushed. 
And I suggest moving this fixture to a common place, also.
Thanks!